docs/pages/enterprise/getting-started.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Teleport Enterprise Quick Start
-description: How to set up and configure Teleport Enterprise for SSH and Kubernetes access
+description: How to set up and configure Teleport Enterprise for SSH, Kubernetes, Databases, Desktops and Web Apps.
```

Comment:
This guide only talks about the SSH service, so I'm not sure Kubernetes, Databases, Desktops belong in the title.